test/lan_items_summary_test.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+require "network/lan_items_summary"
+
+Yast.import "LanItems"
+
+describe Yast::LanItemsSummary do
+  let(:dhcp_maps) do
+    [
+      { "BOOTPROTO" => "dhcp" },
+      { "BOOTPROTO" => "none" },
+      { "IPADDR" => "1.2.3.4", "NETMASK" => "255.255.255.0" }
+    ].freeze
+  end
+
+  let(:items) do
+    {
+      0 => { "ifcfg" => "eth0" },
+      1 => { "ifcfg" => "eth1" },
+      2 => { "ifcfg" => "br0" }
+    }
+  end
+  let(:no_interfaces) { _("Not configured") }
+
+  before do
+    allow(Yast::LanItems).to receive(:Items).and_return(items)
+    allow(Yast::LanItems).to receive(:IsItemConfigured).and_return(true)
+    dhcp_maps.each_with_index do |item, index|
+      allow(Yast::LanItems).to receive(:GetDeviceMap).with(index).and_return(item)
+    end
+  end
+
+  describe "#default" do
+    it "retuns a Richtext summary of the configured interfaces" do
+      expect(subject.default)
+        .to eql "<ul>" \
+                "<li><p>eth0<br>DHCP</p></li>" \
+                "<li><p>eth1<br>NONE</p></li>" \
+                "<li><p>br0<br>STATIC</p></li>" \
+                "</ul>"
+    end
+
+    it "returns Summary.NotConfigured in case of not configured interfaces" do
+      allow(Yast::LanItems).to receive(:IsItemConfigured).and_return(false)
+
+      expect(subject.default).to eql Yast::Summary.NotConfigured
+    end
+  end
+
+  describe "#one_line" do
+    it "returns a plain text summary of the configured interfaces in one line" do
+      expect(subject.one_line).to eql "Multiple Interfaces"
+    end
+
+    context "when there are no configured interfaces" do
+      let(:items) { {} }
+      it "returns 'Not configured'" do
```

Comment:
Sorry, I was thinking in the previous output that was based on Yast::Summary.DevicesList([]), updated!, thnx for check it.